modules/distr-tracing-config-security-ossm-cli.adoc

```diff
@@ -78,16 +78,9 @@ spec:
       readOnly: true
 ----
 +
-. Run the following command to apply your changes, where <jaeger.yaml> is the name of your Jaeger custom resource.
-+
-[source,terminal]
-----
-$ oc apply -n tracing-system -f <jaeger.yaml>
-----
-+
 . Run the following command to watch the progress of the pod deployment:
 +
 [source,terminal]
 ----
-$ oc get pods -n tracing-system -w
+$ oc get pods -n istio-system
```

Comment:
**QE PTAL**
The tracing-system namespace doesn't exist. The other examples in the assembly suggest using `istio-system`.